scripts/ironic-common.sh

```diff
@@ -35,6 +35,12 @@ export IRONIC_USE_MARIADB=${IRONIC_USE_MARIADB:-false}
 
 get_provisioning_interface()
 {
+    # Skip provisioning interface detection if requested (e.g., for exporter)
+    if [[ "${SKIP_PROVISIONING_INTERFACE:-false}" == "true" ]]; then
```

Comment:
```suggestion
    if [[ "${SKIP_PROVISIONING_INTERFACE}" == "true" ]]; then
```
If implemented as suggested in common.sh